docs/libcurl/symbols-in-versions

```diff
@@ -689,6 +689,9 @@ CURLSSLBACKEND_SCHANNEL         7.34.0
 CURLSSLBACKEND_WOLFSSL          7.49.0
 CURLSSLOPT_ALLOW_BEAST          7.25.0
 CURLSSLOPT_NO_REVOKE            7.44.0
+CURLSSLSET_OK                   7.56.0
+CURLSSLSET_UNKNOWN_BACKEND      7.56.0
+CURLSSLSET_TOO_LATE             7.56.0
```

Comment:
I should also draw attention to these lines: it is assumption on my part that this PR could make it into cURL 7.56.0...